<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDeleteDialog</class>
 <widget class="QDialog" name="SettingDeleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Checking time (Every day)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCheckingHour">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>54</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>23</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCheckingMinute">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>54</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Minute</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Keeping term</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>GB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>days</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxImagePrioritySpace">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Priority space</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxImageKeepingSpace">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxImageKeepingMonths">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>70</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Month</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="checkBoxImageKeepMonths">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxImageKeepDays">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="spinBoxImageKeepingDays">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>40</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>File</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Image file name with path</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetImageFileNames">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>251</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonImageSubLine">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>61</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonImageAddFileName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>61</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Folder</string>
    </attribute>
    <widget class="QListWidget" name="listWidgetImageFolderNames">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>251</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonImageSubFolder">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>61</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonImageAddFolderName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>61</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Folder name list</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Keeping remaining space</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
